lib/reporting.rb

```diff
@@ -115,6 +121,10 @@ def mods
     end
   end
 
+  def with_notes_percentage
```

Comment:
Is it possible to DRY things up? Thinking about https://github.com/DFE-Digital/early-years-foundation-recovery/pull/648